chainstate/src/detail/ban_score.rs

```diff
@@ -99,6 +99,7 @@ impl BanScore for ConnectTransactionError {
             // Even though this is an invariant, we consider it a violation to be overly cautious
             ConnectTransactionError::SerializationInvariantError(_) => 100,
             ConnectTransactionError::TimeLockViolation => 100,
+            ConnectTransactionError::TokensError(tokens_error) => tokens_error.ban_score(),
```

Comment:
For brevity, please change `tokens_error` to `err`, just like everything else.

Reply:
Done